src/main/java/com/powsybl/openloadflow/ac/outerloop/IncrementalTransformerVoltageControlOuterLoop.java

```diff
@@ -213,7 +213,9 @@ public OuterLoopStatus check(OuterLoopContext context, Reporter reporter) {
                 double halfTargetDeadband = getHalfTargetDeadband(voltageControl);
                 if (Math.abs(diffV) > halfTargetDeadband) {
                     controlledBusesOutsideOfDeadband.add(controlledBus.getId());
-                    List<LfBranch> controllers = voltageControl.getMergedControllerElements();
+                    List<LfBranch> controllers = voltageControl.getMergedControllerElements().stream()
+                            .filter(b -> !b.isDisabled() && b.isVoltageController())
```

Comment:
"b.isVoltageController()" is not necessary?

Reply:
It was exactly my opinion but I put this because we have this everywhere else... I have to remove that condition for the other outer loops too?